lib/versioning/gradle/index.ts

```diff
@@ -17,7 +18,7 @@ export const urls = [
   'https://docs.gradle.org/current/userguide/single_versions.html#version_ordering',
 ];
 export const supportsRanges = true;
-export const supportedRangeStrategies = ['pin'];
+export const supportedRangeStrategies: RangeStrategy[] = ['pin'];
```

Comment:
This seems strange, but I suppose at worst it's a docs error

Reply:
https://github.com/renovatebot/renovate/blob/main/lib/versioning/gradle/index.ts#L194-L197

I think that is right. 🤷‍♂️